recipes/cobra/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: cobra
+  version: "0.4.0b6"
+
+source:
+  git_url: https://github.com/opencobra/cobrapy.git
```

Comment:
That won't work. We want releases in Bioconda. If there is not release, you still need to trigger a rebuild by increasing the version number from time to time.

But here, there are releases (and you also use the correct version number). So instead of pulling from git, you can just use the release tarballs published [here](https://github.com/opencobra/cobrapy/releases).